BookingService/BookindService.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BookingService.Application;
+using BookingService.Application.DTOs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookindService.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase

```

Comment:
Please add some test controller to verify that your authorization works correctly, with `[Authorize(Roles = "User")`a